src/main/resources/com/typesafe/sbt/packager/archetypes/systemloader/systemv/start-rpm-template

```diff
@@ -86,7 +86,7 @@ start() {
 
 stop() {
     echo -n $"Stopping $prog: "
-    killproc -p $PIDFILE $prog
+    killproc -p $PIDFILE -d ${{kill_timeout}} $prog
```

Comment:
https://gist.github.com/edouard-lopez/4d1c1d007156b662c64e#file-functions-centos-sh-L284 - this is for `functions.centos.sh`
https://bash.cyberciti.biz/guide//etc/init.d/functions - here search for `killproc` - it has line:
```
Usage: killproc [-p pidfile] [ -d delay] {program} [-signal]
```